client/landing/gutenboarding/components/domain-picker-button/index.tsx

```diff
@@ -63,7 +63,7 @@ const DomainPickerButton: FunctionComponent< Props > = ( {
 						className="domain-picker-button__popover"
 						focusOnMount={ false }
 						noArrow
-						onClickOutside={ handleClose }
+						onClickOutside={ handleClose } // TODO: investigate why clicking outside is ignored
```

Comment:
opened issue to track this: https://github.com/Automattic/wp-calypso/issues/40890